ui/app/templates/partials/status/cluster.hbs

```diff
@@ -132,22 +135,20 @@
           </li>
         </ul>
       </nav>
-      {{#if (has-permission 'metrics' routeParams='requests')}}
+      {{#if ( and (has-permission 'metrics' routeParams='requests') (not cluster.dr.isSecondary) auth.currentToken)}}
         <hr />
         <nav class="menu">
           <div class="menu-label">
             Metrics
           </div>
           <ul class="menu-list">
             <li class="action">
-            {{#if activeCluster.unsealed}}
```

Comment:
It's hard for me to get a good grasp of this, but! This looks like one of those changes which is basically 'I just wrapped something in an `if` but git thinks I changed the world' 😄  

The only thing that jumps out at me as 'why was this changed?' is this removal of `activeCluster.unsealed` here. I thought I'd just check with you to make sure you meant to remove this just incase.

Reply:
nice catch! you're right that this is related to matthew's comment. i opted to remove this check and instead [check for an authenticated user](https://github.com/hashicorp/vault/pull/7997/files#diff-2b7fa667fa149e97c987bb886b1934c9R138) since that handles the case where vault is sealed as well. this way we hide the entire "Metrics" navigation item too -- not just the "HTTP Requests" link.